includes/page_top.php

```diff
@@ -2,7 +2,7 @@
     require_once $APPLICATION_DIR.'/includes/application.php';
     $page_request = $_GET['page_request'] ?? \FirstAide\Router::INDEX;
     $query = $_GET['query'] ?? '';
-    $page = FirstAide\Router::getPage($page_request, $query);
+    $page = FirstAide\Router::getPage($UserObj, $page_request, $query);
```

Comment:
@npari I have added the commit message for this file to explain why has this been changed. These function parameters have been restructured to perform mockery.